cmd/bootstrap/cmd/finalize.go

```diff
@@ -139,7 +139,9 @@ func finalize(cmd *cobra.Command, args []string) {
 			filepath.Join(flagOutdir, model.DirnameExecutionState),
 			serviceAccountPublicKey,
 			parseChainID(flagRootChain).Chain(),
-			fvm.WithInitialTokenSupply(value))
+			fvm.WithInitialTokenSupply(value),
+			fvm.WithMinimumStorageReservation(fvm.DefaultMinimumStorageReservation),
```

Comment:
@janezpodhostnik seems the loader is broken without these options, is this an intended behaviour?

Reply:
The storage fees can be disabled, but that happens here: https://github.com/onflow/flow-go/blob/47fade35d97d492742003afb9293ab63c8915bb3/cmd/scaffold.go#L551. If that is set to false the two storage settings are not needed. 

However when I was trying to address this in the integration tests, I didn't find a good point to set that to false, so I added similar code there. I'm sorry I didn't notice this point as well.